### PR TITLE
#trivial MOBILE-967 Fix empty json value into DocumentField

### PR DESCRIPTION
**Pull request by mclomello**
@victorg1991 @dgarciasarai

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java [hunk 1]
```diff
@@ -20,142 +20,155 @@
...
+		} else if (isUploadFailed()) {
+			valid = false;
+		}
+
+		return valid;
+	}
+
+	private boolean isEmptyJson(String stringValue) {
+		try {
+			if (new JSONObject(stringValue).keys().hasNext()) {
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java]: A json with keys is an empty Json?

Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java]: I think I'm missing some context here

Comment (mclomello) [reply android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java]: Victor yeah that logic was wrong. But I rebuild the solution using another approach.

Comment (mclomello) [reply android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java]: @victorg1991 I'll create a test for that

**Comment by victorlaerte** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorlaerte** [comment]
LGTM :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**